### PR TITLE
Change the changelog link to link to prettier/prettier

### PR DESCRIPTION
**Pull request by j-f1**
Ref https://github.com/prettier/prettier/pull/3562#discussion_r158610668.

### CODE REVIEW

File: website/static/playground.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ window.onload = function() {
             prettierVersion;
         } else {
           link.href =
-            "https://github.com/j-f1/forked-prettier/blob/master/CHANGELOG.md#" +
+            "https://github.com/prettier/prettier/blob/master/CHANGELOG.md#" +
             prettierVersion;
```
Comment (ikatyang) [inline website/static/playground.js]: We should omit `.` in this case.

Not working: [`https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1.9.2`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1.9.2)
Working: [`https://github.com/prettier/prettier/blob/master/CHANGELOG.md#192`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#192)

Comment (j-f1) [reply website/static/playground.js]: D’oh! Fixed in https://github.com/prettier/prettier/pull/3564/commits/975aac1e8c9495cc5c8482bff5c368e81f0d3889.

File: website/static/playground.js [hunk 2]
```diff
@@ -96,8 +96,8 @@ window.onload = function() {
...
         } else {
           link.href =
-            "https://github.com/j-f1/forked-prettier/blob/master/CHANGELOG.md#" +
-            prettierVersion;
+            "https://github.com/prettier/prettier/blob/master/CHANGELOG.md#" +
+            prettierVersion.replace(/\./g, '');
```
Comment (ikatyang) [inline website/static/playground.js]: image

Comment (lipis) [reply website/static/playground.js]: we should change some default values :)

**Comment by nurfarma** [comment]
[Obat Pembesar Alat Vital](https://www.kampungobat.com/)
Now as you have done many commits please follow best practices and make the name appropriate and provide a deployment link!

**Pull request merged**